Sources/RibbonKit/List/RibbonListView.swift

```diff
@@ -87,6 +87,13 @@ public class RibbonListView: UIView {
         collectionView.register(RibbonListReusableHostView.self, forSupplementaryViewOfKind: "header")
         return collectionView
     }()
+    
+    #if os(iOS)
+    /// An array of unsupported orientations
+    ///
+    /// This property will be used to prevent header reload for unsupported UIDeviceOrientation
+    public var unsupportedOrientations: [UIDeviceOrientation] = []
```

Comment:
I think the name of this property does not precisely describe what it's doing. Only a second look at the documentation comment revelas its purpose.